hoot-hadoop/src/main/cpp/hoot/hadoop/paint-nodes/PaintNodesMapper.cpp

```diff
@@ -31,8 +31,8 @@ namespace hoot
 PP_FACTORY_REGISTER(pp::Mapper, PaintNodesMapper)
 
 PaintNodesMapper::PaintNodesMapper()
+  : _initialized(false)
 {
-  _context = NULL;
```

Comment:
The base class of `PaintNodesMapper` actually "owns" `_context` and takes care of initialization.